app/views/results/_result_stat.html.erb

```diff
@@ -1,22 +1,17 @@
 <div class="row">
-  <div class="col-12">
-    <h5 style="text-align: center"> Results </h5>
-    <div id="most-yes-result">
-      <h6>Who answered the most <strong>Yes's</strong></h6>
-      <% @total_yes_tally.each do |id, count|%>
-      <% if count == @total_yes_tally.values.max %>
-        <p><strong> <%= Player.find(id[0]).nickname %></strong></p>
-      <% end %>
-    <% end %>
+  <div class="col-12 mt-1">
+    <h4 style="text-align:center;">Results</h4>
+  </div>
+
+  <div class="col-12 d-flex justify-content-center">
+    <div id="most-yes-result" style="width:45%;">
+      <h5>Most <strong>Yes's</strong></h5>
+      <p><%= cl_image_tag @yes_player.user.avatar.key, size: "30x30" %><strong> <%= @yes_player.nickname %> </strong></p>
     </div>
 
-    <div id="most-no-result">
-    <h6> Who answered the most <strong> No's </strong></h6>
-    <% @total_no_tally.each do |id, count|%>
-      <% if count == @total_no_tally.values.max %>
-        <p><strong> <%= Player.find(id[0]).nickname %></strong></p>
-      <% end %>
-    <% end %>
+    <div id="most-no-result" style="width:45%;">
+      <h5>Most <strong> No's </strong></h5>
+      <p><%= cl_image_tag @no_player.user.avatar.key, size: "30x30" %><strong> <%= @no_player.nickname %> </strong></p>
```

Comment:
This no longer take into account if players are have matching yes/no count. We need to make `@yes_players`/`@no_players` an array of players that has values matching with the max value in the hash.

Reply:
That is so weird! I just transferred the code from view to the controller to avoid querying in the view, but I haven't changed the code itself... You mean this only return the first player with the matching number of counts? Should I use where instead??

Reply:
Ok I found a way to put back several players :) I also added an unless array.blank? in the view so that we don't get an error in case nobody answers yes or no!

![Screen Shot 2022-03-11 at 9 33 58](https://user-images.githubusercontent.com/68413600/157778920-bf6fd3ad-66d1-46c8-8f2b-1dd48602deae.png)

Reply:
Thbx, will take a look wheb I get to Le Hub.